tests/foreman/cli/test_contentview.py

```diff
@@ -4476,6 +4476,92 @@ def test_positive_inc_update_no_lce(self):
         self.assertIn(
             '1.1', [cvv_['version'] for cvv_ in content_view['versions']])
 
+    @tier2
+    def test_positive_incremental_update_propagate_composite(self):
+        """Incrementally update a CVV in composite CV with
+        `propagate_all_composites` flag set
+
+        :BZ: 1288148
+
+        :id: 97f7c34d-0b89-49ca-ae2a-65a4552789b8
+
+        :Steps:
+
+            1. Create and publish CV with some content
+            2. Create composite CV, add previously created CV inside it
+            3. Publish composite CV
+            4. Create a puppet repository and upload a puppet module into it
+            5. Incrementally update the CVV with the puppet module with
+               `propagate_all_composites` flag set to `True`
+
+        :expectedresults:
+
+            1. The incremental update succeeds with no errors
+            2. New incremental CVV contains new puppet module
+            3. New incremental composite CVV contains new puppet module
+
+        :CaseLevel: Integration
+        """
+        yum_repo = make_repository({'product-id': self.product['id']})
+        Repository.synchronize({'id': yum_repo['id']})
+        content_view = make_content_view({
+            'organization-id': self.org['id'],
+            'repository-ids': yum_repo['id'],
+        })
+        ContentView.publish({'id': content_view['id']})
+        content_view = ContentView.info({'id': content_view['id']})
+        self.assertEqual(len(content_view['versions']), 1)
+        cvv = ContentView.version_info({
+            'id': content_view['versions'][0]['id']})
+        self.assertEqual(len(cvv['puppet-modules']), 0)
+        comp_content_view = make_content_view({
+            'component-ids': cvv['id'],
+            'composite': True,
+            'organization-id': self.org['id'],
+        })
+        ContentView.publish({'id': comp_content_view['id']})
+        comp_content_view = ContentView.info({'id': comp_content_view['id']})
```

Comment:
definitely, one assertion is missed after that line, otherwise that line doesn't make sense
probably you wanted to check that  comp_cv has one version and zero puppets